supported-smartphones.md

```diff
@@ -1,72 +1,100 @@
-	* In options, select Advertising Extensions and in Primary and Secondary PHY, select LE Coded (Long Range). Leave the other fields default.
-4. On the Master, run the Advertisement by toggling the switch. Select Until manually turned off and No maximum options.
+Suppose the previous test did not eliminate the device from Long Range support.
+In that case, it is now necessary to test the target device’s basic receiving capability.
+At the moment, this information cannot be read from the OS information and must be verified by using another device that supports the Long Range.
+
+1. Prepare the Device Under Test (DUT) for the Long Range receiving support by running [nRF Connect for Mobile](https://play.google.com/store/apps/details?id=no.nordicsemi.android.mcp).
+  * In the Settings menu of the app, select Scanner and set the Scanning period to 5 minutes
+2. Use another device (Master) that passed the Elimination Criteria and run [nRF Connect for Mobile](https://play.google.com/store/apps/details?id=no.nordicsemi.android.mcp).
+	* Alternatively: use any other device that supports Long Range advertisements (might be Remote ID add-on or Bluetooth development kit - e.g., [nRF52840 DK](https://www.nordicsemi.com/Software-and-Tools/Development-Kits/nRF52840-DK)).
+3. On the Master, go to the Advertiser tab and create a New advertising packet.
+	* In options, select Advertising Extensions and in both Primary and Secondary PHY, select LE Coded (Long Range).
+	* Press the "Add Record" and select "Complete Local Name".
+	* Set the Interval to 160 (	resulting in 100 ms).
+	* Set the Tx power to -1 (the maximum).
+	* Leave the other fields at default.
+4. On the Master, run the Advertisement by toggling the switch. Select "Until manually turned off" and "No maximum options".
 	* Alternatively: Start the add-on device or run the [Bluetooth Long Range sample](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/bluetooth/central_hr_coded/README.html).
 5. On the DUT, run the scanning in the Scanner tab and try to find the Master device.
-	* If the Master is close to the DUT, it should have significantly better RSSI (around -40 dBm).
-6. If the Master device can be seen, verify in detail that the Advertising type has value LE Coded. 
-
-Please note that this test says nothing about the possible power-saving features of the phone or Bluetooth Long Range scanning duty cycle. If the test passes, we can only be sure that the DUT somehow receives the Long Range messages, but we don’t know how reliable. Some advanced testing should further verify reliability.
+  * It will have the device name of the Master.
+	* If the Master is close to the DUT, it should have significantly better RSSI than other devices in the list.
+	* Click the found device and verify that the Advertising type is Advertising Extension and that both the Primary and Secondary PHYs are LE Coded.
+	* Drag the device to the right in order to reveal the RSSI chart.
```

Comment:
Before the user swipes to the right to reveal the RSSI chart, he could add Master to the favorites (by clicking on the blue device icon on the left). Then he can set Filter to "Only favorites", so he will be able to inspect just one device in the RSSI chart.

Reply:
Great. Will add.

Reply:
I will add a screenshot showing this, so it is clear for the user